common/changes/office-ui-fabric-react/fix-contextual-menu-scrollbar_2019-05-16-21-22.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "office-ui-fabric-react",
+      "comment": "[ContextualMenu] Render persistent contextual menus with correct scrollbar in Edge.",
```

Comment:
(fwiw: same comment as in your other PR :P)

Reply:
Yeah we have this format:

ContextualMenu: <what's different that the user can expect.>

These go into the release notes, so you should think about how the user will read them and understand what behavior is different. Example:

ContextualMenu: When menus are rendered with the `persistMenu` flag, the scrollbar now renders correctly in Edge.

Reply:
you can use backticks and general markdown in the string as needed.